src/ansys/mapdl/core/_commands/graphics/labeling.py

```diff
@@ -0,0 +1,1247 @@
+        return self.run(command, **kwargs)
+
+    def number(self, nkey: int | str = "", **kwargs):
+        r"""Specifies whether numbers, colors, or both are used for displays.
+
+        Mechanical APDL Command: `/NUMBER <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_NUMBER.html>`_
+
+        Parameters
+        ----------
+        nkey : int or str
+            Numbering style:
+
+            * ``0`` - Color (terminal dependent) the numbered items and show numbers.
+
+            * ``1`` - Color the numbered items. Do not show the numbers.
+
+            * ``2`` - Show the numbers. Do not color the items.
+
+            * ``-1`` - Do not color the items or show the numbers. For contour plots, the resulting display will
+              vary (see below).
+
+        Notes
+        -----
+
+        .. _s-NUMBER_notes:
+
+        Specifies whether numbers, colors, or both are used for numbering displays ( :ref:`pnum` ) of nodes,
+        elements, keypoints, lines, areas, and volumes.
+
+        Shading is also available for terminals configured with more than 4 color planes ( :ref:`show` ).
+        Color automatically appears for certain items and may be manually controlled (off or on) for other
+        items.
+
+        When you suppress color ( ``NKEY`` = -1) your contour plots will produce different results,
+        depending on your graphics equipment. For non-3D devices (X11, Win32, etc.) your contour plot will
+        be white (no color). For 3D devices, such as OpenGL, the resulting display will be in color.
+
+        The following items are automatically given discrete colors: Boundary condition symbols ( :ref:`pbc`
+        ), curves on graph displays, and distorted geometry on postprocessing displays. Contour lines in
+        postprocessing displays are automatically colored based upon a continuous, rather than a discrete,
+        spectrum so that red is associated with the highest contour value. On terminals with raster
+        capability ( :ref:`show` ), the area between contour lines is filled with the color of the higher
+        contour.
+
+        Explicit entity colors or the discrete color mapping may be changed with the :ref:`color` command.
+
+        This command is valid in any processor.
+        """
+        command = f"/NUMBER,{nkey}"
+        return self.run(command, **kwargs)
+
+    def gformat(self, ftype: str = "", nwidth: str = "", dsignf: str = "", **kwargs):
+        r"""Specifies the format for the graphical display of numbers.
+
+        Mechanical APDL Command: `/GFORMAT <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_GFORMAT.html>`_
+
+        Parameters
+        ----------
+        ftype : str
+            FORTRAN format types (G is the default if this field is left blank.)
+
+            * ``G`` - G ``xx``. ``yy``. ``xx`` and ``yy`` are described below.
+
+            * ``F`` - F ``xx``. ``yy``
+
+            * ``E`` - E ``xx``. ``yy``
```

Comment:
We might need to fix this locally:

<img width="573" alt="image" src="https://github.com/user-attachments/assets/36684003-2083-482d-9106-68a2fa1bbc56" />

Reply:
This format comes from the original documentation. It will not be easy to change it the way you would like.